gulpfile.js

```diff
@@ -1,5 +1,18 @@
 'use strict';
 
+const fs = require('fs');
+
+// -----------------------------------------------------------------------------
+// Configuration
+// -----------------------------------------------------------------------------
+
+const SassInput = './assets/scss/styles.scss';
+const SassOutput = './public/css';
+const autoprefixerOptions = { browsers: ['last 2 versions', '> 5%', 'Firefox ESR'] };
+const config = JSON.parse(fs.readFileSync('./package.json'));
+global.vfName = config.vfConfig.vfName;
```

Comment:
can you talk me through this bit later?

Reply:
Yep. It's more about the child theme support and thinking that maintenance would be easier when we can get VF-Core and the child theme to use as close to identical config files as possible.